src/posts/building-a-restish-json-api-with-play-scala.md

```diff
@@ -0,0 +1,15 @@
+---
+title: 'Building a REST-ish JSON API with Play Scala'
+date: 2019-09-13T12:00:09.441Z
+tags:
+  - post
+layout: layouts/post.njk
+videoId: XNRYB2ppkeI
+
+---
+
+You spend your days writing code, most likely a RESTful backend or a client consuming said backend. You’ve done this before, and you’re going to do this again. In the heat of the project it’s easy to forget to note all the things you did well and the things you could improve for next time. 
+
+In this talk I’ll list some of the best practices that have stuck with me on how to build a RESTful backend. We’ll cover general advice applicable to any technology choice as well as some practical examples using the Play Framework and Scala programming language.
+
+Presenter  - Oskar Ehnström
```

Comment:
Not unified syntax here, some of these use colon and some a dash between presenter and the name.

Reply:
Never mind it hasn't been unified earlier either. Could do a cleanup later.